TourDeOpole/TourDeOpole/Views/AddTripView.xaml

```diff
@@ -1,13 +1,88 @@
 ﻿<?xml version="1.0" encoding="utf-8" ?>
 <ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
-             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml" xmlns:viewmodels="clr-namespace:TourDeOpole.ViewModels" x:DataType="viewmodels:AddTripViewModel"
              x:Class="TourDeOpole.Views.AddTripView"
-             Title="AddTripView">
+             Shell.NavBarHasShadow="False"
+             Title="Dodaj trase"
+             Shell.TabBarIsVisible="False">
+
     <ContentPage.Content>
-        <StackLayout>
-            <Label Text="Welcome to AddTripView!"
-                VerticalOptions="CenterAndExpand" 
-                HorizontalOptions="CenterAndExpand" />
-        </StackLayout>
+
+        <Grid  RowDefinitions="75,75,100,*,50,70" 
+              ColumnDefinitions="40,*,40"
+              BackgroundColor="White">
+            <Frame Grid.ColumnSpan="3"
+                   Grid.Column="0"
+                   Grid.Row="0"
+                   BackgroundColor="{DynamicResource Primary} "/>
+            <Frame CornerRadius="90" 
+                   HorizontalOptions="Center"
+                   VerticalOptions="Center"
+                   IsClippedToBounds="True" 
+                   Grid.Column="1"
+                   Grid.Row="0"
+                   Grid.RowSpan="2"
+                   HeightRequest="100" 
+                   WidthRequest="100" 
+                   BorderColor="LightGray">
+                <Image Source="AddAPhoto.png" 
+                       Margin="10"/>
+            </Frame>
+            <ImageButton Grid.Row="2" 
```

Comment:
To na pewno może być zwykły image bo nie mamy żadnej funkcjonalności po kliknięciu. A może wgl zwykły label?